tests/v0/tests_unit/test_session/test_read.py

```diff
@@ -95,3 +95,27 @@ def test_read_file_metadata(self) -> None:
             )
         )
         assert instances_ids == expected
+
+
+def test_read_rdf_instances_with_named_graph(tmp_path) -> None:
+    """Test reading RDF instances into a named graph"""
+
+    ttl_file = tmp_path / "test.ttl"
+    ttl_file.write_text(
+        """
+        @prefix ex: <http://example.org/> .
+        ex:subject1 ex:predicate1 "value1" .
+        ex:subject2 ex:predicate2 "value2" .
+    """
+    )
+
+    neat = NeatSession()
+
+    # Test with string named_graph
+    issues = neat.read.rdf.instances(ttl_file, named_graph="my_test_graph")
+    assert not issues.has_errors
+
+    expected_graph_uri = NAMED_GRAPH_NAMESPACE["my_test_graph"]
```

Comment:
nit: Since you are here testing the expected graph uri, I think it is better to write the full URL instead of using `NAMED_GRAPH_NAMESPACE`. Changing that constant should cause a test failure imo.